README.md

```diff
@@ -1,9 +1,71 @@
 # terraform-scripts
 
 This repository holds our common Terraform Open edX deployment scripts.
+They can be used to provision and manage the infrastructure required by an Open edX instance on AWS.
 
-Currently it actually contains two different ways to provision AWS resources
+### TODOS:
+- [ ] Write usage documentation
+- [ ] Expand the conventions section below with more details and examples
+
+## Architecture map
+
+See [this document](https://docs.google.com/drawings/d/1s2hd6hJSKo1a-eqp6rI2kF-afACZXlK3f3ymptZjV6M/edit?usp=sharing).
+
+## Conventions
+### Resource naming
+
+Every resource provisioned by Terraform should be created using the following naming schema:
+
+```
+`{client_shortname}-{environment}-{resource}-{additional_info}`
+```
+
+Examples:
+- `opencraftx-production-appserver-1` - Appserver from the production environment of client `OpenCraftX`
+- `example-stage-alb` - Alb resource for client `Example`
+
+To ease the migration and import process, the resources were renaming implies on changing credentials
```

Comment:
Typo:
~~the resources were renaming implies on changing credentials...~~ → 
for resources where renaming implies changing credentials...